tests/e2e/scripts/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM ubuntu:20.04
+ARG EXTERNAL_ENCODED_VPN
+ARG VPN_ENCODED_LOGIN
+
+RUN apt-get update && \
+    apt-get install -y curl bridge-utils iputils-ping openvpn openssh-client && \
```

```suggestion
    DEBIAN_FRONTEND=noninteractive apt-get install -y \
        bridge-utils \
        curl \
        iputils-ping \
        openvpn openssh-client \
        && \
```

It is easier to maintain and avoid duplicate entries if the packages are split on separate lines and sorted alphabetically.